### DISCUSSION TITLE
How to add ammonia reaction mechanism in PeleC

### DISCUSSION THREAD

**Discussion by hzw-zju-2023** [AMReX-Combustion/PeleC#738] [Q&A]
I want to perform numerical simulation of liquid ammonia in PeleC, so I need the reaction mechanism of ammonia. Since I have not been in contact with PeleC for a long time, I am not familiar with how to convert the ammonia mechanism in chemkin format into the mechanism in PeleC. Would you like some familiar friends to help me convert? I have attached my chemkin file, thanks
[chemkin.zip](https://github.com/AMReX-Combustion/PeleC/files/13816788/chemkin.zip)

**Comment by marchdf** [comment]
Hi, thanks for reaching out. Here are instructions to do so: https://amrex-combustion.github.io/PelePhysics/Ceptr.html#converting-chemkin-files. If this doesn't answer your question, please let us know and we can improve the documentation.
